### ISSUE TITLE
Command line parser not work from version 1.6.2-beta (included)

### ISSUE THREAD

**Issue by valinha** [microsoft/ApplicationInspector#493]
**Describe the bug**
From version 1.6.2-beta the command line parser output does not work (shows nothing). In version 1.6.1-beta it works correctly.

**To Reproduce**
1. Run ApplicationInspector without args

Note: I execute this from docker:

dockerfile:
```
FROM mcr.microsoft.com/dotnet/sdk:6.0
RUN dotnet tool install --global Microsoft.CST.ApplicationInspector.CLI --version 1.6.2-beta
ENV PATH="${PATH}:/root/.dotnet/tools"
```

**Expected behavior**
The help output should appear, example:

ApplicationInspector.CLI 1.6.1-beta+6c40a50fb6
© Microsoft Corporation. All rights reserved.

ERROR(S):
  No verb selected.

  analyze        Inspect source directory/file/compressed file (.tgz|zip) against defined characteristics

  tagdiff        Compares unique tag values between two source paths

  exporttags     Export the list of tags associated with the specified rules. Does not scan source code.

  verifyrules    Verify custom rules syntax is valid

  packrules      Combine multiple rule files into one file for ease in distribution

  help           Display more information on a specific command.

  version        Display version information.

**Screenshots**
With version 1.6.1-beta:
<img width="876" alt="image" src="https://user-images.githubusercontent.com/24825183/183466012-12dddc0f-9fdd-460e-b71c-8637168cd999.png">

With version >= 1.6.2-beta:
<img width="388" alt="image" src="https://user-images.githubusercontent.com/24825183/183466199-61ef46ea-a37e-43b2-84ee-0ae0fc47112d.png">

**Operating Environment (please complete the following information):**
 - Application Inspector Version: [run AppInspector with `--version`], >= 1.6.2-beta
 - OS: [e.g. Windows 11 21H2 22000.613, MacOS 13.0.1, Linux Ubuntu 20.04]. MacOs using docker
 - Browser: (if an issue with HTML report) [e.g. Edge, Chrome, Safari]
 - Browser Version: (if an issue with HTML report) [e.g. 100]

**Additional context**
dockerfile:
```
FROM mcr.microsoft.com/dotnet/sdk:6.0
RUN dotnet tool install --global Microsoft.CST.ApplicationInspector.CLI --version 1.6.2-beta
ENV PATH="${PATH}:/root/.dotnet/tools"
```

**Comment by gfs** [comment]
I'll check this today. Likely an unintended side effect of specifying options to changing to case insensitive enum parsing in the area.

**Comment by gfs** [comment]
This was caused by switching from Parser.Default to an instance of Parser with arguments specified required for enum case insensitivity (confidence and severity arguments). I'll merge the fix today that manually does the printing that was automatically being done by the Default parser on failure.

**Comment by gfs** [comment]
I think this is resolved in 1.6.8-beta, which is on nuget now. Thanks for the report.

![image](https://user-images.githubusercontent.com/98900/183536844-d469b149-642f-4414-aed9-d490c690d2b0.png)
